Dockerfile

```diff
@@ -34,6 +34,13 @@ RUN apt-get update \
     && sed 's/#.*//' /tmp/apt-run.txt | xargs apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
+# Read configure from docker image folder
+COPY testing/eastern_cropmask/results/gm_mads_two_seasons_ml_model_* /crop-mask/testing/eastern_cropmask/results/
```

Comment:
Do we have to copy the folders? Can't we copy just the `model`, `td`, and `aez` files themselves?

Reply:
the model all have datetime. Do you want to only copy one datetime?

Reply:
now only specific model file is copied.